modules/fhir-client/test/blaze/fhir_client_test.clj

```diff
@@ -312,3 +312,25 @@
         ::anom/category := ::anom/fault))))
 
 
+(deftest basic-auth-client
+  (testing "Authenticator is set"
```

Comment:
testing is a macro and should be indented with two spaces.

Reply:
I see two spaces. Am I missing something?

Reply:
not the testing macro itself. the let macro should idented only two spaces more than the testing macro

Reply:
Got it, fixed.